spec/features/reservations/host_can_view_reservations_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe "Viewing reservations" do
+  let!(:traveler)    { create(:user, first_name: "Traveler name") }
+  let!(:host)        { create(:user, first_name: "Host name") }
+  let!(:couch)       { create(:couch, host: host) }
```

Comment:
@tmikeschu Yes, you're correct!  I did it this way because I was getting confused with the connections, and this way read more clearly for me - though it's on the verbose side.